app/views/builds/show.html.haml

```diff
@@ -19,6 +19,8 @@
       Compare to last green build
 
   .actions
+    - if @build.succeeded?
+      = button_to "Resend Status", resend_status_repository_build(@build), :method => :post, :form_class => "resend-status"
```

Comment:
The button text should be a little more descriptive of what it is doing. As a kochiku user I would be guessing as to what will happen when I click this button.

Ideal text would be "Sync status with Bitbucket" or "Sync status with Github" where the scm name is taken from the `type` column in `application.yml`

Reply:
huh resend status seemed fairly clear to me 🤷‍♂️